### PR TITLE
Zd sci 512 global team switch

### PR DESCRIPTION
**Pull request by ZmagoD**
https://biosistemika.atlassian.net/browse/SCI-512

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1125,6 +1125,9 @@ en:
         team_name_label: "Team name"
         team_name_help: "Team name is required in order to create your own team."
     settings:
+      changed_org_flash: "You are working on %{team} now!"
+      changed_org_error_flash: "Something get wrong! Try again later."
```
Comment (mz3944) [inline config/locales/en.yml]: Why you no spell correct!!!

File: app/assets/stylesheets/themes/scinote.scss [hunk 2]
```diff
@@ -554,6 +554,33 @@ a[data-toggle="tooltip"] {
   }
 }
 
+// Global team switch
+#team-switch {
```
Comment (mz3944) [inline app/assets/stylesheets/themes/scinote.scss]: če češ...

**Comment by mz3944** [comment]
screenshot from 2016-10-17 08 52 28

If team name is long enough, then the navigation bar items get stacked horizontally. Fix it so that they remain vertical.

**Comment by ZmagoD** [comment]
Should I truncate the team name here?

File: app/views/projects/index.html.erb [hunk 3]
```diff
@@ -132,9 +111,10 @@
 </div>
 <% end %>
 
-<% if @projects_by_orgs.length > 0 %>
-  <% @projects_by_orgs.each do |org, projects| %>
-    <%= render partial: "projects/index/org_projects", locals: {org: org, projects: projects} %>
-  <% end %>
+<% if current_organization.projects.length> 0 %>
```
Comment (mz3944) [inline app/views/projects/index.html.erb]: .length >

**Comment by mz3944** [comment]
If team has no projects, that the team name isn't shown until at least one project is present.

screenshot from 2016-10-17 10 53 08
screenshot from 2016-10-17 10 53 21

**Comment by mz3944** [comment]
The team switch seems to work nicely. After fixing the above mentioned issues, you can merge.

**Comment by ZmagoD** [comment]
fixed

**Comment by ZmagoD** [comment]
I will close this and made a new PR to modular-tasks branch  #219

**Pull request closed without merging**